lib/update/cluster/phases/bootstrap.go

```diff
@@ -277,6 +300,7 @@ type updatePhaseBootstrapLeader struct {
 	existingClusterConfig []byte
 	// updateManifest specifies the manifest of the update application
 	updateManifest schema.Manifest
+	kubernetesOperation
```

Comment:
As we discussed offline, just put `client *kubernetes.Clientset` here, you don't really need to embed kubernetesOperation.